folder/README.md

```diff
@@ -6,4 +6,27 @@ We are about to make a commit without best practices.
 
 hi
 
-goodbye
\ No newline at end of file
+goodbye
+
+docs
+
+
+repositories, branches, commits, tags
+
+# created the repo locally
+local repository -> "remote" repository (git add remote <name> <URL>)
+
+# created the repo remotely
+git clone <URL>
```

Comment:
I find this section vague. Please consider adding a verbal description of what these commands are actually doing.

Reply:
```suggestion
# created the repo remotely
git clone <URL>  # deeply copy and download all repository metadata
```